resources/builder.inc.sh

```diff
@@ -290,61 +357,152 @@ _builder_trim() {
   printf '%s' "$var"
 }
 
+#
+# Expands an in-repo-relative path to a repo-relative path. A path starting with
+# `/` is expected to be relative to repo root, not filesystem root. Otherwise,
+# it's relative to current script path, not current working directory. The
+# returned path will not have a prefix `/`, and will be relative to
+# `$KEYMAN_ROOT`. Assumes realpath is installed (brew coreutils on macOS).
+#
+_builder_expand_relative_path() {
+  local path="$1"
+  if [[ "$path" =~ ^/ ]]; then
+    echo "${path:1}"
+  else
+    realpath --canonicalize-missing --relative-to="$KEYMAN_ROOT" "$THIS_SCRIPT_PATH/$path"
```

Comment:
not sure why i'm hitting this now, but for what it's worth `realpath` on mac doesn't have those opts 

> `     realpath [-q] [path ...]`